trio/lowlevel.py

```diff
@@ -16,6 +16,7 @@
 from ._core import (
     cancel_shielded_checkpoint,
     Abort,
+    RaiseCancelT,
```

Comment:
This import seems extraneous?

Reply:
Just removing it from there makes `test_core_is_properly_reexported` angry, so it's mostly a question of whether an internal `TypeVar` should be exposed in the API or not. It's not incredibly complex or anything, but might be of use to users? Unsure, I made it internal now but easy to change back.

Reply:
Ah didn't notice it would make a test angry, nvm then. If we're going to make a type alias we might as well go all the way and make it public